otlp/scripts/generate-modules.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
```

Comment:
I'd have been tempted to write this in Haskell as an executable in the package, but this is also good :)